src/content/tools/pub/dependencies.md

```diff
@@ -248,15 +258,16 @@ The Dart community uses semantic versioning<sup id="fnref:semver"><a
 href="#fn:semver">1</a></sup>.
 
 You can express version constraints using either _traditional syntax_
-or _caret syntax_. Both syntaxes specify a range of compatible versions.
+or _caret syntax_ starting with Dart 2.19.
+Both syntaxes specify a range of compatible versions.
 
 The traditional syntax provides an explicit range like `'>=1.2.3 <2.0.0'`.
 The caret syntax provides an explicit starting version `^1.2.3`
 
 ```yaml
 environment:
-  # This package must use a 2.x version of the Dart SDK starting with 2.14.
-  sdk: '>=2.14.0 < 3.0.0'
+  # This package must use a 2.x version of the Dart SDK starting with 2.19.
+  sdk: '^2.19.0'
```

Comment:
Hmm consider changing this one to `^3.2.0` as SDK constraints `>= 2.12.0` have one-off behavior of actually supporting 3.x versions of the SDK. That just avoids any potential confusion here.

```suggestion
  # This package must use a 3.x version of the Dart SDK starting with 3.2.
  sdk: ^3.2.0
```